cmd/kubeadm/app/phases/copycerts/testutil_umask.go

```diff
@@ -0,0 +1,27 @@
+// +build darwin linux
```

Comment:
there's no `// +build unix` (I hallucinated it I swear) and we don't currently actively support platforms other than these + windows, with these two being the only known platforms we support that do. 

I didn't find many comparable examples in the existing codebase, mostly code that targets linux or windows specifically (for e.g. kubelet).